test/helper/common.bash

```diff
@@ -469,5 +469,19 @@ function _print_mail_log_for_id() {
   _run_in_container grep -F "${MAIL_ID}" /var/log/mail.log
 }
 
+# A simple wrapper for netcat (`nc`). This is useful when sending
+# "raw" e-mails or doing IMAP-related work.
+#
+# @param ${1} = the file that is given to `nc`
+# @param ${1} = custom parameters for `nc` [OPTIONAL] (default: 0.0.0.0 25)
+function _nc_wrapper() {
+  local FILE=${1:?Must provide name of template file}
```

Comment:
> If you do keep `.txt` omitted, you might as well do the same here?

Deferring to a later PR

> Likewise moving the `/tmp/docker-mailserver-test/` path prefix, since there's only two relative paths `nc/file.txt` emails/nc_raw/file.txt`

👍🏼

Reply:
Actually, a simple search-and-replace with Regexes did the trick